services/QuillLMS/client/app/bundles/Shared/components/shared/postNavigationBanner.tsx

```diff
@@ -44,6 +47,7 @@ export const PostNavigationBanner = ({ tagText, primaryHeaderText, secondaryHead
         </div>
       </div>
       <img alt={icon.alt} className="banner-icon" src={icon.src} />
+      {closeIconSrc && <button aria-label={closeAria} className="interactive-wrapper close-button" onClick={handleCloseCard} type="button"><img alt="" src={closeIconSrc} /></button>}
```

Comment:
I might use `handleCloseCard` as the conditional here rather than `closeIconSrc`, just bc in the instance where we have one without the other (which I know shouldn't happen anyway) I would care more about the button having some functionality than having an image.